packages/core/lib/console.js

```diff
@@ -87,7 +87,10 @@ class Console extends EventEmitter {
   provision() {
     let files;
     try {
-      files = fse.readdirSync(this.options.contracts_build_directory);
+      const unfilteredFiles = fse.readdirSync(
+        this.options.contracts_build_directory
+      );
+      files = unfilteredFiles.filter(file => file.match(".json") != null);
```

Comment:
Just one nit-picky thing...can you use `!==` instead of `!=`?

Reply:
For sure, done!